external/llvm-project/mlir/test/Dialect/Tosa/tosa-partition.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: mlir-opt --split-input-file --tosa-partition %s -verify-each=0 -o - | FileCheck %s
+// RUN: mlir-opt --split-input-file --tosa-partition='trailing-only=false' %s -verify-each=0 -o - | FileCheck %s
```

Comment:
Why verify-each=0?

Reply:
Because I blindly copied the options from the test I started with.  Seriously, though, I was just following what was there.